packages/nestjs-user/src/user.types.ts

```diff
@@ -1,4 +1,11 @@
+import { PasswordStrengthTransformOptionsInterface } from '@concepta/nestjs-common';
+import { PasswordStrengthEnum } from '@concepta/nestjs-password';
+
 export enum UserResource {
   'One' = 'user',
   'Many' = 'user-list',
 }
+
+export type PasswordStrengthTransform = (
+  options: PasswordStrengthTransformOptionsInterface,
```

Comment:
options parameter should be optional
```suggestion
  options?: PasswordStrengthTransformOptionsInterface,
```